extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -116,7 +122,8 @@ require([
 				withCredentials: true
 			},
 		}).done(function (data) {
-			var threads = processData(data._embedded.threads, requestData.upvoteRequestUrl);
+			var threads = processData(data._embedded.threads, requestData.upvoteRequestUrl),
+				i;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'i' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedVariable)